changelog/v1.19.0-beta9/pin-linter.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+  - type: NON_USER_FACING
+    resolvesIssue: false
+    description: >-
+      Pin golangci-lint-action to `6.4.1` to avoid breaking changes in `6.5.0`
```

Comment:
Do we intend to update as part of this work/soon in the future and this is a temp fix, or 6.4 is the plan longer-term?